src/app/ui/upload/upload.component.ts

```diff
@@ -37,6 +40,7 @@ enum UploadState {
   templateUrl: './upload.component.html',
   styleUrls: ['./upload.component.scss']
 })
+@Injectable()
```

Comment:
Why did you add it? a component shouldn't be injectable..

Reply:
I looked online for using httpClient and that was the example, with an injectable, now I understand it is not related to the httpClient, but only there for the example. I will remove it.